src/api.js

```diff
@@ -173,3 +173,20 @@ export const MOCK_EVENT = {
 	visibility: 'public',
 	yes_rsvp_count: 23
 };
+
+// Mock category sourced from https://www.meetup.com/meetup_api/docs/find/topic_categories/
+export const MOCK_CATEGORY = {
```

Comment:
Maybe you can leave this as MOCK_CATEGORY, but also add a `export const MOCK_TOPIC_CATEGORIES = [MOCK_CATEGORY]`, that can be imported directly by the tests.